### PR TITLE
1650: Adjust self service texts

### PR DESCRIPTION
**Pull request by f1sh1918**
### Short description

Finalize texts except hints and error messages (separate issue)

### Proposed changes

- adjust texts
- add help dialog
- adjust basic dialog for mobile to have a close text button to have consistent styling for all dialogs

### Side effects

- none

### Testing

- Check the texts according to:
[design](https://www.figma.com/design/e4cHjTPVjrAVoYYLp1zfC5/KoblenzPass-ShareFile?node-id=1-2924&node-type=frame&t=aiQRP7ITTiDN42II-0
)
- test help dialog
- test aktenzeichen dialog

### Resolved issues

Fixes: #1650

### CODE REVIEW

File: administration/src/bp-modules/self-service/CardSelfServiceActivation.tsx [hunk 1]
```diff
@@ -11,20 +11,28 @@ const Container = styled('div')`
...
+      KoblenzPass PDF
+    </StyledIconTextButton>
+    <InfoText>
+      Um Ihren KoblenzPass auf dem aktuellen Gerät zu aktivieren, klicken Sie bitte auf „Pass jetzt aktivieren“. <br />
+      <br />
+      <b>Wichtig: </b>Die KoblenzPass-App muss dafür installiert sein (siehe Schritt 2 von 3).
+    </InfoText>
     <ActionButton href={deepLink} variant='contained' size='large'>
-      Pass aktivieren
+      KoblenzPass jetzt aktivieren
```
Comment (ztefanie) [inline administration/src/bp-modules/self-service/CardSelfServiceActivation.tsx:35]: Please use `APP_NAME` constant instead

File: administration/src/bp-modules/self-service/CardSelfServiceActivation.tsx [hunk 2]
```diff
@@ -11,20 +11,28 @@ const Container = styled('div')`
   flex-direction: column;
 `
 
+const StyledIconTextButton = styled(IconTextButton)`
+  color: #131314;
```
Comment (ztefanie) [inline administration/src/bp-modules/self-service/CardSelfServiceActivation.tsx:15]: Please move this to name constant that is reusable.

File: administration/src/bp-modules/self-service/CardSelfServiceForm.tsx [hunk 3]
```diff
@@ -64,7 +64,7 @@ const CardSelfServiceForm = ({
   return (
     <>
       <Container key={card.id}>
-        <FormGroup label='Vorname Nachname'>
+        <FormGroup label='Vorname Name'>
```
Comment (ztefanie) [inline administration/src/bp-modules/self-service/CardSelfServiceForm.tsx:67]: weird, but ok :D

File: administration/src/bp-modules/self-service/components/IconTextButton.tsx [hunk 4]
```diff
@@ -7,7 +7,9 @@ export const IconTextButton = styled('button')`
   background-color: transparent;
   border: none;
   cursor: pointer;
-  font-size: 16px;
+  font-size: 14px;
   align-items: center;
   margin-bottom: 12px;
+  color: #595959;
```
Comment (ztefanie) [inline administration/src/bp-modules/self-service/components/IconTextButton.tsx:13]: Please move this to a named constant

**Review by ztefanie — COMMENTED** [review]
Did not recheck every text in the designs.

The Help dialog would look nicer on desktop if the width would be smaller

**Review by ztefanie — APPROVED** [review]

File: administration/src/bp-modules/self-service/CardSelfServiceActivation.tsx (hunk 1 above)
Comment (steffenkleinle) [reply administration/src/bp-modules/self-service/CardSelfServiceActivation.tsx:35]: Same for other occurrences :)

**Review by steffenkleinle — APPROVED** [review]

**Comment by f1sh1918** [comment]
> Did not recheck every text in the designs.
> 
> The Help dialog would look nicer on desktop if the width would be smaller
> 
> 

Hm I think a text line break should also help. I will adjust that. Thx for the hint

File: administration/src/bp-modules/self-service/components/IconTextButton.tsx (hunk 4 above)
Comment (f1sh1918) [reply administration/src/bp-modules/self-service/components/IconTextButton.tsx:13]: i think it makes more sense to define a colors file at some point and define there everything. This is sth that should be created and used project wide.
I will create a ticket for that

**Pull request merged**